transport/sender.go

```diff
@@ -0,0 +1,28 @@
+package transport
+
+import (
+	"io"
+	"time"
+)
+
+type MultiSender struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type MultiSender should have comment or be unexported